src/main/java/io/jenkins/plugins/sprp/YAML_FlowDefinition.java

```diff
@@ -146,23 +152,30 @@ public FlowExecution create(FlowExecutionOwner owner, TaskListener listener,
         return refSpecs;
     }
 
-    /*
-
-    Bit-bucket generated following two similar refspecs:
-    refs/heads/abhishekg1128/readmemd-edited-online-with-bitbucket-1529079381853:refs/remotes/origin/PR-1
-    refs/heads/master:refs/remotes/upstream/master
-
-    And GitHub generates following two similar refspecs in single string"
-    +refs/pull/3/head:refs/remotes/origin/PR-3 +refs/heads/master:refs/remotes/origin/master
-
-    */
-    private String getBranchName(String[] refSpecsArray){
+    /**
+     * Bit-bucket generates following two similar refSpecs for pull request:
+     * {@code refs/heads/abhishekg1128/readmemd-edited-online-with-bitbucket-1529079381853:refs/remotes/origin/PR-1}
+     * {@code refs/heads/master:refs/remotes/upstream/master}
```

Comment:
There are two bitbucket implementations, 1) Bitbucket cloud (bitbucket.org) and 2) Bitbucket server. I guess these refs are for the Bitbucket cloud implementation?

On Bitbucket server, I see these references:
```
$ git ls-remote
From http://martin@localhost:7990/scm/spid/orange.git
f2a85eb1e8f6b47f94186d8e251c0bd984d04e4b	HEAD
bb1caf949f0300a82cd4242cc8b3d7469305a1a1	refs/heads/bugfix/exec-hello
f2a85eb1e8f6b47f94186d8e251c0bd984d04e4b	refs/heads/master
8fc7aafee8e7314ea98be9ea3f39a4f28306630e	refs/pull-requests/1/from
e30f42c437b9456b4f0f15e6f3be4690b3fdde8a	refs/pull-requests/1/merge
```

Reply:
References mentioned by you are shown in the terminal like this. RefSpecs is the mapping of remote references and local references. When we try to get refSpecs in the code it will be in the format that I mentioned in the comment for both bitbucket cloud and bitbucket server.